cpp/daal/include/data_management/data/internal/numeric_table_sycl_homogen.h

```diff
@@ -571,9 +571,49 @@ class DAAL_EXPORT SyclHomogenNumericTable : public SyclNumericTable
 };
 /** @} */
 
+/**
+ * Converts numeric table with arbitrary storage layout to SYCL homogen numeric table of the given type
+ * \param[in]  src               Pointer to numeric table
```

Comment:
It's not actually pointer. Add doc for status parameter

Reply:
Done